### ISSUE TITLE
Android WebView with empty Source breaks rendering

### ISSUE THREAD

**Issue by jhndnnqsc** [dotnet/maui#9392]
### Description

The following XAML will render as a white square on Android ( ok on WIndows and iOS ). If I either assign a Source in code or hit the invisible GO button the WebView and other controls render fine.

```
<?xml version="1.0" encoding="utf-8" ?>
<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="MauiApp3.MainPage">
  <ScrollView>
    <Grid>
      <Grid.RowDefinitions>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="*"/>
      </Grid.RowDefinitions>
      <Button Clicked="Button_Clicked" Text="GO"></Button>
      <Editor Grid.Row="1" x:Name="txt" Text="https://www.google.com" HorizontalOptions="FillAndExpand"/>
      <WebView Grid.Row="2" x:Name="wv"></WebView>
    </Grid>
  </ScrollView>
</ContentPage>
```

### Steps to Reproduce

1. Create new MAUI App
2. Paste above code into MainPage.xaml
3. Run on Android emulator
4. Page shows blank

### Version with bug

6.0.400

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 12.1

### Did you find any workaround?

No

**Comment by Eilon** [comment]
Related/dupe? #9751

**Comment by rmarinho** [comment]
Fixed by #9916
